octopoes/octopoes/models/ooi/dns/records.py

```diff
@@ -155,3 +156,33 @@ class DNSPTRRecord(DNSRecord):
     @classmethod
     def format_reference_human_readable(cls, reference: Reference) -> str:
         return f"{reference.tokenized.address.address} -> {reference.tokenized.hostname.name}"
+
+
+class CAATAGS(Enum):
+    ISSUE = "issue"
+    ISSUEWILD = "issuewild"
+    IODEF = "iodef"
+    CONTACTEMAIL = "contactemail"
+    CONACTPHONE = "contactphone"
+    ISSUEVMC = "issuevmc"
+    ISSUEMAIL = "issuemail"
+    
+    def __str__(self):
+        return self.value
+
+
+class DNSCAARecord(DNSRecord):
+    object_type: Literal["DNSCAARecord"] = "DNSCAARecord"
+    dns_record_type: Literal["CAA"] = "CAA"
+
+    # https://datatracker.ietf.org/doc/html/rfc8659#name-canonical-presentation-form
+    # An unsigned integer between 0 and 255.
+    flags: Optional[int] = None
+
+    # A non-zero-length sequence of ASCII letters and numbers in lowercase.
+    tag: CAATAGS = None
+
+    # The Value field, expressed as either (1) a contiguous set of characters
+    # without interior spaces or (2) a quoted string.
+    value: str = None
```

Comment:
The same goes for this one. Based on the natural key attributes I can guess that this is a required field, so therefore it should be just `value: str`